### ISSUE TITLE
New page

### ISSUE THREAD

**Issue by AliAtaMoussa** [givanz/VvvebJs#114]
Hello, 

I am having a problem when I add a new page, it doesn't show on the left menu under pages. I checked the code, and there is some errors in the builder.js, anybody faced this issue?

Thanks

**Comment by oleteacher** [comment]
Yes, do a search for "add page" on closed. For example: https://github.com/givanz/VvvebJs/issues/49

**Comment by AliAtaMoussa** [comment]
Can you please give me an example for the steps of naming? I am trying #49 steps and not working.Thanks

**Comment by oleteacher** [comment]
Sorry, I cannot. Never got it working and never got any help to do so. https://github.com/givanz/VvvebJs/issues/49#issuecomment-471330476

Was just pointing out the issue I had, not a solution, so sorry.

**Comment by AliAtaMoussa** [comment]
No worries, thanks!

**Comment by nikhilkhadgi** [comment]
https://github.com/givanz/VvvebJs/issues/49#issuecomment-594493319
